plugins/catalog-backend-module-terraform-cloud/src/processors/TerraformCloudProcessor.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LocationSpec, Entity } from '@backstage/catalog-model';
+import {
+  CatalogProcessor,
+  CatalogProcessorEmit,
+  results
+} from '@backstage/plugin-catalog-backend';
+import fetch from 'cross-fetch';
+import { Config } from '@backstage/config';
+import  { Logger } from 'winston';
+
+export class TfCloudReaderProcessor implements CatalogProcessor {
+    constructor(private readonly config: Config, private readonly logger: Logger) {}
```

Comment:
When reading config during instantiation we'd typically use the `fromConfig` pattern, see for example https://github.com/backstage/backstage/blob/658131a2900928cf4d020f7f2bedcdf529bb5555/plugins/catalog-backend/src/ingestion/processors/AzureDevOpsDiscoveryProcessor.ts#L43

Reply:
ok - I've updated to use this pattern, appreciate any and all feedback / pointers on where I've gone wrong here!